src/components/CurrencyInputPanel/CurrencyInputPanel.styles.ts

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components'
+import transparentize from 'polished/lib/color/transparentize'
+
+import { breakpoints } from '../../utils/theme'
+import { ReactComponent as DropDown } from '../../assets/images/dropdown.svg'
+
+export const InputRow = styled.div<{ selected: boolean }>`
+  ${({ theme }) => theme.flexRowNoWrap}
+  margin-bottom: 8px;
+  align-items: center;
+`
```

Comment:
```suggestion
export const InputRow = styled.div`
  ${({ theme }) => theme.flexRowNoWrap}
  margin-bottom: 8px;
  align-items: center;
`
```

I think this can be removed since selected prop is not used inside.

Reply:
Fixed. You can take a look again :)